### PR TITLE
Fix/63/Hero image not fitting at different zooms

### PR DESCRIPTION
**Pull request by AmirAgassi**
[Issue #63 ](https://github.com/LaurierHawkHacks/Landing/issues/63)

🔍 What's Included

1. Updated HeroSection component for responsive image scaling and text alignment.

🔍 Not included in the issue, but relevant problems were identified and fixed:

2. Adjustments in StatSection for more responsive text and better positioning at different screen widths.

3. Improved FooterSection with responsive hero image and adaptable footer layout.

📈 Before & After:

63_comparison

📁 Files Affected:
- `src/components/sections/HeroSection.tsx`
- `src/components/sections/StatSection.tsx`
- `src/components/sections/FooterSection.tsx`

### CODE REVIEW

File: src/components/sections/Hero.section.tsx [hunk 1]
```diff
@@ -52,7 +52,12 @@ const HeroSection: React.FC = () => {
                     REGISTER
                 </button>
                 <img
-                    className="-mt-2"
+                    style={{
+                        width: '100%',
+                        display: 'block',
+                        marginLeft: 'auto',
+                        marginRight: 'auto'
+                    }}
```
Comment (juancwu) [inline src/components/sections/Hero.section.tsx:60]:
```suggestion
                    className="w-full block mx-auto"
```

Just want to stick to Tailwind where ever possible.

File: src/components/sections/Footer.section.tsx [hunk 2]
```diff
@@ -11,15 +11,20 @@ function FooterSection() {
 
   return (
     <footer className="bg-midnight text-white">
-      <img className="-mt-2" src={HeroIllustration} alt="Hero Illustration" />
-      <div className="mt-[-5.5rem] flex items-center justify-between px-24">
+      <img 
+        style={{ width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto' }}
```
Comment (juancwu) [inline src/components/sections/Footer.section.tsx:15]:
```suggestion
        className="w-full block mx-auto"
```

**Review by juancwu — CHANGES_REQUESTED** [review]

File: src/components/sections/Footer.section.tsx (hunk 2 above)
Comment (AmirAgassi) [reply src/components/sections/Footer.section.tsx]: True!! I've replaced the inline styles with Tailwind CSS classes, tested and it looks good to me!

**Review by juancwu — APPROVED** [review]
lgtm

**Pull request merged**